### PR TITLE
Nfdiv 1672 & Nfdiv 1757

### PR DESCRIPTION
**Pull request by velimd**
### JIRA link ###
https://tools.hmcts.net/jira/browse/NFDIV-1672
https://tools.hmcts.net/jira/browse/NFDIV-1757

### Change description ###
image
image

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: src/main/steps/applicant1/review-the-application/content.ts [hunk 1]
```diff
@@ -180,15 +192,24 @@ const languages = {
...
 
 export const generateContent: TranslationFn = content => {
-  const translations = languages[content.language](content);
-  const isApplicantAddressNotPrivate = content.userCase.applicant1AddressPrivate !== YesOrNo.YES;
-  const isRespondentAddressNotPrivate = content.userCase.applicant2AddressPrivate !== YesOrNo.YES;
-  const isFinancialOrderYes = content.userCase.applicant1ApplyForFinancialOrder === YesOrNo.YES;
+  const { language, userCase } = content;
+  const translations = languages[language](content);
+  const isApplicantAddressNotPrivate = userCase.applicant1AddressPrivate !== YesOrNo.YES;
+  const isRespondentAddressNotPrivate = userCase.applicant2AddressPrivate !== YesOrNo.YES;
```
Comment (benouaer) [inline src/main/steps/applicant1/review-the-application/content.ts:198]: why use negative, it's confusing

File: src/main/steps/applicant1/review-the-application/template.njk [hunk 2]
```diff
@@ -60,9 +60,20 @@
     <h3 class="govuk-heading-m">{{ subHeading10 }}</h3>
     <p class="govuk-body">{{ line22 }}</p>
   {% endif %}
-  <h3 class="govuk-heading-m">{{ subHeading11 }}</h3>
+  {% if isApplicant2Represented %}
+    <h3 class="govuk-heading-m">{{ subHeading11 }}</h3>
+    {% if not solInfoEntered %}
```
Comment (benouaer) [inline src/main/steps/applicant1/review-the-application/template.njk:65]: why not flip them to avoid negation

**Review by benouaer — APPROVED** [review]

File: src/main/steps/applicant1/review-the-application/content.ts (hunk 1 above)
Comment (velimd) [reply src/main/steps/applicant1/review-the-application/content.ts:198]: tbh, that wasnt my code change, i just changed the variable for userCase but ill make it easier to read

File: src/main/steps/applicant1/review-the-application/template.njk (hunk 2 above)
Comment (velimd) [reply src/main/steps/applicant1/review-the-application/template.njk:65]: i believe i originally had that but i forgot why i flipped it so ill just go with your opinion.

**Review by linusnorton — APPROVED** [review]

**Pull request merged**